pagopa/io-app
# chore(IT Wallet): [SIW-514] Update copy in ITW_DISCOVERY_INFO screen

## Short description
This PR updates the copy in the ITW_DISCOVERY_INFO screen and updates the typography accoring to [this FIGMA page](https://www.figma.com/file/c80PUA6BYANkGPyaqsRBNA/IT-Wallet---Esperienza-in-IO?type=design&node-id=2263-43532&mode=design&t=dDj0EJ5Yv3U8DKMO-0).


## List of changes proposed in this pull request
- Replaces typography with `H6` component as mentioned in the FIGMA.
- Updates copy for the existing sections; 
- Adds a new "where" section with its own copy.

## How to test
Start the app and press on the ITW activation banner. The first screen should be updated with the new copy and typography.
